### PR TITLE
Extended value in Magnifier + Phone DE QWERTZ and FR AZERTY layouts

### PR DESCRIPTION
**Pull request by Herrie82**

### CODE REVIEW

File: plugins/fr/qml/Keyboard_fr_phone.qml [hunk 1]
```diff
@@ -52,59 +56,63 @@ KeyPad {
...
-                CharKey        { id: atKey;    label: "@"; shifted: "@";     anchors.left: symShiftKey.right; }
+                SymbolShiftKey { id: symShiftKey; label: "123"; shifted: "123"; anchors.left: parent.left; }
+                LanguageKey    { id: languageMenuButton;                       anchors.left: symShiftKey.right; }
+                UrlKey         { id: atKey;    label: "@"; shifted: "@";     anchors.left: languageMenuButton.right; }
                 SpaceKey       { id: spaceKey;                               anchors.left: atKey.right; anchors.right: urlKey.left; noMagnifier: true }
-                UrlKey         { id: urlKey; label: ".fr"; extended: [".ca", ".cd", ".ci", ".ch", ".be", ".ht"]; anchors.right: dotKey.left; }
-                CharKey        { id: dotKey;      label: "."; shifted: ".";  anchors.right: enterKey.left; }
-                ReturnKey      { id: enterKey;                               anchors.right: parent.right }
+                UrlKey         { id: urlKey; label: ".com"; shifted: ".com"; extended: [".com", ".de", ".net", ".org", ".edu", ".gov"]; anchors.right: dotKey.left; }
```
Comment (Tofee) [inline plugins/fr/qml/Keyboard_fr_phone.qml]: You sure ? in the "fr" phone layout ?

File: plugins/fr/qml/Keyboard_fr_phone.qml [hunk 2]
```diff
@@ -52,59 +56,63 @@ KeyPad {
...
-            CharKey { label: "n"; shifted: "N"; extended: ["ñ"]; extendedShifted: ["Ñ"] }
-            CharKey { label: "ç"; shifted: "Ç"; }
-            BackspaceKey {}
+            ShiftKey {id: shiftKey; anchors.left: parent.left; }
+            CharKey { id: wKey; anchors.left: shiftKey.right; width: UI.keyWidth * 10/8; label: "w"; shifted: "W"; }
+            CharKey { id: xKey; anchors.left: wKey.right; width: UI.keyWidth * 10/8; label: "x"; shifted: "X"; extended: ["x", "Rec", "Mute"]; extendedShifted: ["X", "Rec", "Mute"]; }
+            CharKey { id: cKey; anchors.left: xKey.right; width: UI.keyWidth * 10/8; label: "c"; shifted: "C"; extended: ["c", "ç", "ć", "©", "¢"]; extendedShifted: ["C", "Ç", "Ć", "©", "¢"]; }
+            CharKey { id: vKey; anchors.left: cKey.right; width: UI.keyWidth * 10/8; label: "v"; shifted: "V"; }
+            CharKey { id: bKey; anchors.left: vKey.right; width: UI.keyWidth * 10/8; label: "b"; shifted: "B"; }
+            CharKey { id: nKey; anchors.left: bKey.right; width: UI.keyWidth * 10/8; label: "n"; shifted: "N"; extended: ["n", "ñ", "ń"]; extendedShifted: ["N", "Ñ", "Ń"]; }
```
Comment (Tofee) [inline plugins/fr/qml/Keyboard_fr_phone.qml]: If that would make a 9/8 ratio, I'd like to add a " ' " key here, which is very common in french sentences.

File: plugins/fr/qml/Keyboard_fr_phone.qml (hunk 1 above)
Comment (Herrie82) [inline plugins/fr/qml/Keyboard_fr_phone.qml]: Do you need it in the URL/email layout? I can add it there if needed as shifted :P

File: plugins/fr/qml/Keyboard_fr_phone.qml (hunk 2 above)
Comment (Herrie82) [inline plugins/fr/qml/Keyboard_fr_phone.qml]: OK, you want a shifted value as well or should be ' as well?

File: plugins/fr/qml/Keyboard_fr_phone.qml (hunk 1 above)
Comment (Tofee) [inline plugins/fr/qml/Keyboard_fr_phone.qml]: No, I mean that it's strange to see that you replaced a ".fr" UrlKey with a ".de" UrlKey in the fr layout...

File: plugins/fr/qml/Keyboard_fr_phone.qml (hunk 2 above)
Comment (Tofee) [inline plugins/fr/qml/Keyboard_fr_phone.qml]: The shifted value ccould be whatever you like. Maybe " ?

File: plugins/fr/qml/Keyboard_fr_phone.qml (hunk 1 above)
Comment (Herrie82) [inline plugins/fr/qml/Keyboard_fr_phone.qml]: Oops sloppy C&P. You's say you'd use .com or .fr more in France?

Comment (Tofee) [inline plugins/fr/qml/Keyboard_fr_phone.qml]: .com would come first, I guess

Comment (Herrie82) [inline plugins/fr/qml/Keyboard_fr_phone.qml]: Should be solved with https://github.com/Herrie82/webos-keyboard/commit/3a0168934b7d322e45da512d06de0e09b542be10

File: qml/keys/AnnotatedKey.qml [hunk 3]
```diff
@@ -202,7 +202,7 @@ Item {
         anchors.bottom: buttonImage.top
         width: key.width + Units.gu(UI.magnifierHorizontalPadding)
         height: key.height + Units.gu(UI.magnifierVerticalPadding)
-        text: keyLabel.text
+        text: UI.currentShiftState === "SHIFTED" ? annotationLabel.text : keyLabel.text
```
Comment (Tofee) [inline qml/keys/AnnotatedKey.qml]: This is incorrect is currentShitState === "CAPSLOCK"; better use valueToSubmit, has it necessarily matches what the user types

**Pull request merged**